automatic/virtualbox/update.ps1

```diff
@@ -1,5 +1,6 @@
 import-module au
 import-module .\..\..\extensions\extensions.psm1
+Import-Module $PSScriptRoot\..\..\scripts\au_extensions.psm1
```

Comment:
This line is not needed, this psm1 module will already be loaded automatically when calling `update_all.ps1` script in the root repository.

I understand it can be helpful for debugging purposes, but please remove it from the committed file.

Reply:
Addressed in 1e8e4ec59d649dab83c122d946862d89ed0bd60c.